dist/angular-dynamic-layout.js

```diff
@@ -153,20 +160,24 @@
 
   angular
     .module('dynamicLayout')
-    .directive('layoutOnLoad', ['$rootScope', layoutOnLoad]);
+    .directive('layoutOnLoad', ['$rootScope', '$timeout', layoutOnLoad]);
 
   /*
    * Directive on images to layout after each load
    */
-  function layoutOnLoad($rootScope) {
+  function layoutOnLoad($rootScope, $timeout) {
 
     return {
-        restrict: 'A',
-        link: function(scope, element) {
-          element.bind('load error', function() {
-            $rootScope.$broadcast('layout');
+      restrict: 'A',
+      link: function(scope, element) {
+        element.bind('load error', function() {
+          var timeoutId;
```

Comment:
@tristanguigue 
I did get after parsing dist files this error:
`Uncaught ReferenceError: timeoutId is not defined`

Thats why I added it there.
The code itself which is running with $timeout is not from my side.
It is from this merge you did: https://github.com/tristanguigue/angular-dynamic-layout/commit/d162d32be51435e3d129920cc59befe1c3e3a523

I have to say I have no clue whats about.
Can someone tell where it should be set properly to fix the original error?